utils/src/main/scala/za/co/absa/enceladus/utils/types/EnceladusDateParser.scala

```diff
@@ -18,21 +18,25 @@ package za.co.absa.enceladus.utils.types
 import java.text.SimpleDateFormat
 import java.sql.Date
 import java.sql.Timestamp
+import java.time.format.DateTimeFormatter
 
 /**
   * Enables to parse string to date and timestamp based on the provided format
   * Unlike SimpleDateFormat it also supports an epoch ("epoch", "milliepoch"...) formatting strings
   * @param pattern  the formatting string, in case it's an epoch format the values wil need to be convertible to Long
   */
 case class EnceladusDateParser(pattern: Format) {
-  private val simpleDateFormat: Option[SimpleDateFormat] = if (pattern.isEpoch) {
+  //TODO use Option.when with Scala 2.13
```

Comment:
Please don't commit TODO's, FIXME, etc. to code. Use issues or other platforms available.